system/indy-node-tests/TestLedgerSuite.py

```diff
@@ -591,16 +604,17 @@ async def test_pool_restart(
         # SETUP---------------------------------------------------------------------------------------------------------
         trustee_did, _ = get_default_trustee
         # --------------------------------------------------------------------------------------------------------------
-        req = await ledger.build_pool_restart_request(trustee_did, action, _datetime)
-        res = json.loads(await ledger.sign_and_submit_request(pool_handler, wallet_handler, trustee_did, req))
-        res = {k: json.loads(v) for k, v in res.items()}
-        assert all([v['op'] == 'REPLY' for k, v in res.items()])
+        res = await eventually(
+            send_pool_restart, pool_handler, wallet_handler, trustee_did, action, _datetime
+        )
+        assert all([json.loads(v)['op'] == 'REPLY' for k, v in res.items()])
 
     @pytest.mark.parametrize('timeout', [5, 3600])
     @pytest.mark.parametrize('justification', [None, random_string(1), random_string(1000)])
     @pytest.mark.parametrize('reinstall', [False, True])
     @pytest.mark.parametrize('force', [False, True])
-    @pytest.mark.parametrize('package', ['indy-node', 'sovrin'])
+    # @pytest.mark.parametrize('package', ['indy-node', 'sovrin'])
+    @pytest.mark.parametrize('package', ['indy-node']) # unable to install sovrin package, it is installed via dpkg -i in Dockerfile
```

Comment:
This should only be a problem is you are attempting to run the payment tests AND the Sovrin package is not installed.